tests/conftest.py

```
@@ -390,60 +388,21 @@ def _make(
-                """Record device removal and return the configured removal result.
-
-                Args:
-                    *args: Positional removal arguments accepted for API compatibility.
-                    **kwargs: Keyword removal arguments accepted for API compatibility.
-                """
-                self.removed = True
-                return self._remove_result
-
-        return _FakeReg()
+        registry = MagicMock()
+        registry.async_get_device.side_effect = lambda *args, **kwargs: (
```

Ruff (0.15.21)

[warning] Unused lambda argument: `args`

(ARG005)